src/cli/commands-cn/info.js

```
@@ -17,17 +17,18 @@ module.exports = async (config, cli, command) => {
   // Start CLI persistance status
   cli.sessionStart('Initializing', { timer: false });
 
-  await utils.login();
-
-  if (runningTemplate(process.cwd())) {
+  if (runningTemplate(process.cwd(), true)) {
     return infoAll(config, cli);
```

Can we confirm there is not user info required in the method `infoAll`, as previously we have this issue when moving login after the logic. 

Also, I think we can simply check `isInvalidPath(root)` and throw the invalid error if is not a serverless project or template folder. With this, we can just add an extra check without changing our existing logic.  Current issue is that

* A folder contains multiple serverless projects (obviously not a template as well) considered as a template folder during the run time.